script/app-coverage-report.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const path = require('path');
+const Table = require('cli-table');
+
+const camelCase = str =>
+  // Convert string to camel case
+  str
+    .replace(
+      /(?:^\w|[A-Z]|\b\w)/g,
+      (word, index) => (index === 0 ? word.toLowerCase() : word.toUpperCase()),
+    )
+    .replace(/-|\s/g, '');
+
+const printCoverage = coverageResults => {
+  // Create a new table with headers
+  const coverageTable = new Table({
+    head: ['Application', 'Lines', 'Functions', 'Statements', 'Branches'],
+  });
+
+  // Add each app coverage result to the table
+  Object.keys(coverageResults).forEach(key => {
+    if (key !== 'total') {
+      coverageTable.push({
+        [key]: [
+          `${coverageResults[key].lines.pct}%`,
+          `${coverageResults[key].functions.pct}%`,
+          `${coverageResults[key].statements.pct}%`,
+          `${coverageResults[key].branches.pct}%`,
+        ],
+      });
+    }
+  });
+
+  /* eslint-disable no-console */
```

Comment:
If it's just for this line, I think you probably meant `eslint-disable-line`. Or if you wanted for the whole module, I would move it near the top to make it more visible.